helmfile/charts/user-rbac/templates/clusterroles/user-view.yaml

```diff
@@ -6,3 +6,9 @@ rules:
 - apiGroups: [""]
   resources: ["nodes","namespaces","persistentvolumes"]
   verbs: ["get", "watch", "list"]
+- apiGroups: ["metrics.k8s.io"]
+  resources: ["pods","nodes"]
+  verbs: ["list"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["list"]
```

Comment:
Based on our discussion earlier that `list` basically gets them the same privilege as `get`, should we maybe give them `get` as well? Maybe even `watch`?

Reply:
I'm okay with that.

Reply:
:+1: